modules/migration-configuring-mcg.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_3_4/configuring-replication-repository.adoc
+// migration/migrating_4_1_4/configuring-replication-repository.adoc
+// migration/migrating_4_2_4/configuring-replication-repository.adoc
+[id='migration-configuring-mcg_{context}']
+= Configuring a Multi-Cloud Object Gateway storage bucket as a replication repository
+
+You can install the OpenShift Container Storage Operator and configure a Multi-Cloud Object Gateway (MCG) storage bucket as a replication repository on the CAM tool cluster.
+
+[id='installing-the-ocs-operator_{context}']
+== Installing the OpenShift Container Storage Operator
+
+You can install the OpenShift Container Storage Operator from OperatorHub on the CAM tool cluster.
+
+.Procedure
+
+. In the {product-title} web console, click *Administration* -> *Namespaces*.
+. Click *Create Namespace*.
+. Enter `openshift-storage` in the *Name* field and click *Create*.
+. Click *Operators* -> *OperatorHub*.
+. Use *Filter by keyword* (in this case, *OCS*) to find the *OpenShift Container Storage Operator*.
+. Select the *OpenShift Container Storage Operator* and click *Install*.
+. On the *Create Operator Subscription* page, select the `openshift-storage` namespace.
+. Specify your update channel and approval strategy.
+. Click *Subscribe*.
++
+On the *Installed Operators* page, the *OpenShift Container Storage Operator* appears in the *openshift-storage* project with the status *InstallSucceeded*.
```

Comment:
In OCP 4.4 the Status is **Succeeded** once operator is installed successfully

Reply:
Good to hear!